tests/integration_test/core_mqtt_system_test.c

```diff
@@ -226,7 +243,7 @@
  * PUBLISH message and ack responses for QoS 1 and QoS 2 communications
  * with the broker.
  */
-#define MQTT_PROCESS_LOOP_TIMEOUT_MS          ( 700U )
+#define MQTT_PROCESS_LOOP_TIMEOUT_MS          ( 1000U )
```

Comment:
Although, this increase seems insignificant. What if we try increasing the timeout to 10 seconds? Based on the data, the latency can possibly be that long.